### PR TITLE
OAI-46: Added entrypoint for running asgi app

### PR DESCRIPTION
**Pull request by dborowiecki**
Part of [OAI-46](https://openimis.atlassian.net/secure/RapidBoard.jspa?rapidView=28&projectKey=OAI&modal=detail&selectedIssue=OAI-46). Default entrypoint does not support  websocket requests, alternative option allowing using daphne as application server was added.

### CODE REVIEW

File: openIMIS/openIMIS/settings.py [hunk 1]
```diff
@@ -200,22 +202,32 @@ def SITE_ROOT():
...
+if os.environ.get('SIMPLE_DATABASE', 'False') == 'True':
+    # used by AI instance that does not require running database
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.sqlite3',
+            'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        }
+    }
+else:
+    DATABASES = {
```
Comment (dragos-dobre) [inline openIMIS/openIMIS/settings.py:221]: Use NO_DATABASE_ENGINE env var to disable DB.

File: openIMIS/openIMIS/settings.py [hunk 2]
```diff
@@ -200,22 +202,32 @@ def SITE_ROOT():
...
-        'ENGINE': os.environ.get('DB_ENGINE', 'sql_server.pyodbc'),
-        'NAME': os.environ.get('DB_NAME'),
-        'USER': os.environ.get('DB_USER'),
-        'PASSWORD': os.environ.get('DB_PASSWORD'),
-        'HOST': os.environ.get('DB_HOST'),
-        'PORT': os.environ.get('DB_PORT'),
-        'OPTIONS': DATABASE_OPTIONS}
-}
+
+if os.environ.get('SIMPLE_DATABASE', 'False') == 'True':
```
Comment (dragos-dobre) [inline openIMIS/openIMIS/settings.py:212]: Remove this.

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

File: openIMIS/openIMIS/settings.py (hunk 2 above)
Comment (dborowiecki) [reply openIMIS/openIMIS/settings.py]: Removed

File: openIMIS/openIMIS/settings.py (hunk 1 above)
Comment (dborowiecki) [reply openIMIS/openIMIS/settings.py:213]: NO_DATABASE_ENGINE used

**Review by dragos-dobre — APPROVED** [review]

**Pull request merged**